azure-functions-maven-plugin/src/it/0-http-trigger/pom.xml

```diff
@@ -18,7 +18,7 @@
         <functionAppName>maven-functions-it-${timestamp}-0</functionAppName>
         <functionAppRegion>westus</functionAppRegion>
         <stagingDirectory>${project.build.directory}/azure-functions/${functionAppName}</stagingDirectory>
-        <functionResourceGroup>java-functions-group</functionResourceGroup>
```

Comment:
since we used deploymentType default value in integration test, it will naturally test against zip deploy without any change after we change zip deploy as default value.